core/exception/case_compare_spec.rb

```diff
@@ -1,5 +1,35 @@
 require_relative '../../spec_helper'
 
 describe "SystemCallError.===" do
-  it "needs to be reviewed for spec completeness"
+  before :all do
+    @example_errno = 22
+    @example_errno_class = Errno::EINVAL
+  end
+
```

Comment:
Could you add a spec for `Errno::EINVAL.should === Errno::EINVAL.new` which is the simplest case?

Reply:
Will do.

Reply:
What would be the description?

  it "???" do
    Errno::EINVAL.should === Errno::EINVAL.new
  end